PointOfDispensingApp/src/components/VaccinationScreeningComponent.vue

```diff
@@ -1,95 +1,284 @@
-              </v-radio-group>
-            </v-row>
-          </v-col>
-          <v-col cols="6">
-            <v-row no-gutters>
-              <v-radio-group v-model="patientInfoComfirmed" row @change="screeningChecklistUpdate()">
-                  <v-radio
-                    label="Yes"
-                    value="Yes"
-                  ></v-radio>
-                  <v-radio
-                    label="No"
-                    value="No"
-                  ></v-radio>
-              </v-radio-group>
-            </v-row>
-          </v-col>
-        </v-row> 
-        <v-row no-gutters>
-          <v-col cols="6">
-            <v-row no-gutters>
-              <v-radio-group>
-                <template v-slot:label>
-                  <div>Consent form signed?</div>
-                </template>
-              </v-radio-group>
-            </v-row>
-          </v-col>
-          <v-col cols="6">
-            <v-row no-gutters>
-              <v-radio-group v-model="consentFormSigned" row @change="screeningChecklistUpdate()">
-                  <v-radio
-                    label="Yes"
-                    value="Yes"
-                  ></v-radio>
-                  <v-radio
-                    label="No"
-                    value="No"
-                  ></v-radio>
-              </v-radio-group>
-            </v-row>
-          </v-col>
-        </v-row>
-        <v-row no-gutters>
-          <v-col cols="6">
-            <v-row no-gutters>
-              <v-radio-group>
-                <template v-slot:label>
-                  <div>Proper screening completed?</div>
-                </template>
-              </v-radio-group>
-            </v-row>
-          </v-col>
-          <v-col cols="6">
-            <v-row no-gutters>
-              <v-radio-group v-model="screeningCompleted" row @change="screeningChecklistUpdate()">
-                  <v-radio
-                    label="Yes"
-                    value="Yes"
-                  ></v-radio>
-                  <v-radio
-                    label="No"
-                    value="No"
-                  ></v-radio>
-              </v-radio-group>
-            </v-row>
-          </v-col>
-        </v-row>
-        <v-row no-gutters>
-          <v-col cols="6">
-            <v-row no-gutters>
-              <v-radio-group>
-                <template v-slot:label>
-                  <div>VIS fact sheet provided?</div>
-                </template>
-              </v-radio-group>
-            </v-row>
-          </v-col>
-          <v-col cols="6">
-            <v-row no-gutters>
-              <v-radio-group v-model="factSheetProvided" row @change="screeningChecklistUpdate()">
+    <v-row>
+      <v-col cols="10">
+        <v-row>
+          <v-radio-group>
+            <template v-slot:label>
+              <div><span class="red--text"><strong>* </strong></span>
+              <span class="font-weight-medium secondary--text">1. Are you feeling sick today?</span></div>
```

Comment:
Couple things here
1. Can we shift the radio buttons over to the left a little? Basically have the end of them line up with the end of the text boxes above. They're really far over and makes the border look odd
2. Can we shift the sub questions over to the left two? They're really far over and it looks like way too much space. There's also an odd amount of vertical spacing between the questions as well...
3. Let's make the font color of the subtext darker - it was hard to read and I imagine being tired would make that a lot worse

![image](https://user-images.githubusercontent.com/67079030/106233988-6bc94d80-61bd-11eb-9383-6afef507113d.png)

Reply:
This will take some digging and trial-and-error, but I'll see what I can do!  This is the best I could do with the simplest / default component implementation.  I agree that these cosmetic adjustments would be better.

Reply:
I made the radio button text darker, but adjusting the indents of the secondary questions is not looking like a trivial change - is it worth it?  I can bring it up in the meeting later today to see if anyone has other ideas.

Reply:
For now, let's make a card and move on.